File: CodeBackup/spark/WrapperService - Copy/src/main/java/com/tekclover/wms/core/model/enterprise/Warehouse.java
package com.tekclover.wms.core.model.enterprise;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
public class Warehouse { 
	
	private String warehouseId;
	private String companyId;
	private String plantId;
	private String modeOfImplementation;
	private Long warehouseTypeId;
	private String languageId;
	private Boolean inboundQACheck;
	private Boolean outboundQACheck;
	private String zone;
	private String address1;
	private String address2;
	private String city;
	private String state;
	private String country;
	private Long zipCode;
	private String contactName;
	private String designation;
	private String phoneNumber;
	private String eMail;
	private Double length;
	private Double width;
	private Double totalArea;
	private String uom;
	private Long noAisles;
	private Double latitude;
	private Double longitude;
	private String storageMethod;
	private Long deletionIndicator = 0L;
	private String createdBy;
	private Date createdOn = new Date();
	private String updatedBy;
	private Date updatedOn = new Date();

}

File: ALMailam/Code/Project/Web/backup/dev/05-05/src/app/models/warehouse.ts
export interface Warehouse {
    address1?: string;
    address2?: string;
    city?: string;
    companyId?: string;
    contactName?: string;
    country?: string;
    createdBy?: string;
    createdOn?: Date;
    deletionIndicator?: number;
    designation?: string;
    email?: string;
    inboundQACheck?: boolean;
    languageId?: string;
    latitude?: number;
    length?: number;
    longitude?: number;
    modeOfImplementation?: string;
    noAisles?: number;
    outboundQACheck?: boolean;
    phoneNumber?: string;
    plantId?: string;
    state?: string;
    storageMethod?: string;
    totalArea?: number;
    uom?: string;
    updatedBy?: string;
    updatedOn?: Date;
    warehouseId?: string;
    warehouseTypeId?: number;
    width?: number;
    zipCode?: number;
    zone?: string;
}

export interface WarehouseList {
    warehouse?: Warehouse[];
}